aptos-move/vm-genesis/src/lib.rs

```diff
@@ -729,6 +754,40 @@ fn create_and_initialize_validators_with_commission(
     );
 }
 
+fn create_pbo_delegation_pools(
+    session: &mut SessionExt,
+    pbo_delegator_configuration: &[PboDelegatorConfiguration],
+) {
+    let pbo_config_bytes = bcs::to_bytes(pbo_delegator_configuration)
+        .expect("PboDelegatorConfiguration can be serialized");
+    let mut serialized_values = serialize_values(&vec![
+        MoveValue::U64(90), // Delegation % - hardcoded for prototyping
```

Comment:
It would be better to define a appropriately named constant at the top of the file or provide this parameter via config rather than using `90` as a magic number here.

Reply:
Yep, I missed this. I'll add a constant in the next commit. Thanks for the catch 👍

Reply:
Addressed in 44e74f3e81e55dbd58cac2834abaf383ba2eed1a